users/views.py

```diff
@@ -0,0 +1,47 @@
+from .forms import RegisterForm
+from django.contrib.auth import authenticate, login, logout
+from django.shortcuts import render, redirect
+from .forms import LoginForm
+
+
+# home page
+def index(request):
+    return render(request, 'base.html')
+
+
+# registration page
+def register(request):
+    if request.method == 'POST':
+        form = RegisterForm(request.POST)
+        if form.is_valid():
```

Comment:
why you dont handle when form not valid? add validation for this, with notification users about errors